src/features/foods/components/FilterButton.tsx

```diff
@@ -108,93 +123,109 @@ export const FilterButton = ({
    * Process when group filter button is pushed
    * @param event
    */
-  const handleSelectGroup = (event: React.ChangeEvent<HTMLSelectElement>) => {
-    setSelectedGroup(event.target.value);
+  const handleSelectGroup = (value: string) => {
+    setSelectedGroup(value);
+    setSelectedContainer('');
   };
 
   /**
    * Process when container filter button is pushed
    * @param event
    */
-  const handleSelectContainer = (event: React.ChangeEvent<HTMLSelectElement>) => {
-    setSelectedContainer(event.target.value);
+  const handleSelectContainer = (containerId: string) => {
+    setSelectedContainer(containerId);
+    setSelectedGroup(containerIdGroupIdMap[containerId]);
   };
   const [isDrawerOpen, setIsDrawerOpen] = useState(false);
 
   return (
     <DrawerRoot open={isDrawerOpen} onOpenChange={setIsDrawerOpen}>
       <DrawerTrigger asChild>
-        <button className="flex justify-center items-center absolute top-0 right-1 h-12 w-12">
-          <Icon icon={FilterIcon} size={4} className="justify-center" />
+        <button className="flex justify-center items-center absolute top-0 right-0 h-full px-4">
+          <div className="relative">
+            <FilterIcon className="w-3.5" />
+            {isFilterSet && (
+              <div className="absolute -top-[3px] -right-[5px] w-2 h-2 rounded-full bg-danger" />
+            )}
+          </div>
         </button>
       </DrawerTrigger>
       <DrawerContent side="bottom">
         <DrawerHeader>
           <DrawerTitle>Filter Food</DrawerTitle>
```

Comment:
Is Implementing all of food filter drawer, and foodCategory drawer inside of `FilterButton` component is the best way?
I felt it a bit too much responsibility inside of `FilterButton` component.

It might be out of the scope of this PR, but I wanted to hear your point of view on this.

Reply:
It's a bit difficult to explain my opinion on how we could avoid Implementing all of food filter drawer, and foodCategory drawer inside of `FilterButton` , so if you're interested in my specific idea, let's have a chat when you have time! (maybe tomorrow)

Reply:
> I felt it a bit too much responsibility inside of FilterButton component.

I totally agree with it. It should have been split into smaller components in the first place.
I actually attempted to refactor them, but decided not to do in this PR because I thought it would take quite long.

Since it's a `"FilterButton"` component, it would be better to separate all the `DrawerContent` as different components.

Reply:
We will address this matter in https://github.com/genesis-tech-tribe/nishiki-frontend/issues/121